stream_alert/shared/metrics.py

```diff
@@ -84,6 +88,28 @@ class MetricLogger(object):
         },
         ALERT_PROCESSOR_NAME: {},   # Placeholder for future alert processor metrics
         ATHENA_PARTITION_REFRESH_NAME: {},  # Placeholder for future athena processor metrics
+        CLASSIFIER_FUNCTION_NAME: {
+            FAILED_PARSES: (_default_filter.format(FAILED_PARSES),
```

Comment:
I believe we need to remove metrics in `CLASSIFIER_FUNCTION_NAME` from `RULE_PROCESSOR_NAME` below.

Reply:
I will do this when all of the rule processor code is removed - this will happen in one later PR :)